core_contracts/governance/governance.py

```diff
@@ -58,6 +64,84 @@ def getVotersCount(self, name: str) -> dict:
         vote_index = ProposalDB.proposal_id(name, self.db)
         proposal = ProposalDB(var_key=vote_index, db=self.db)
         return {'for_voters': proposal.for_voters_count.get(), 'against_voters': proposal.against_voters_count.get()}
+    
+    @external
+    @only_owner
+    def setMinimumVoteDuration(self, duration: int) -> None:
+        """
+        Set the minimum vote duration.
+
+        Parameters:
+        duration - minimum amount of days a vote has to be active.
+        """
+        self._minimum_vote_duration.set(duration)
+
+    @external(readonly=True)
+    def getMinimumVoteDuration(self) -> int:
+        """
+        Returns the minimum vote duration in days.
+        """
+        return self._minimum_vote_duration.get()
+
+    @external
+    @only_owner
+    def setQuorum(self, quorum: int) -> None:
+        """
+        Set percentage of total baln supply which must participate in a vote 
+        for the vote to be valid.
+
+        Parameters:
+        quorum - percentage of total baln supply required for a vote to be valid.
+        """
+        if not 0 < quorum < 100:
+            revert("Quorum must be between 0 and 100.")
+        self._quorum.set(quorum)
+
+    @external(readonly=True)
+    def getQuorum(self) -> int:
+        """
+        Returns the percentage of total baln supply which must participate in a vote
+        for the vote to be valid.
+        """
+        return self._quorum.get()
+    
+    @external
+    @only_owner
+    def setVoteDefinitionFee(self, fee: int) -> None:
+        """
+        Set the fee for defining votes. Fee in bnUSD.
+        """
+        self._bnusd_vote_definition_fee.set(fee)
+
+    @external(readonly=True)
+    def getVoteDefinitionFee(self) -> int:
+        """
+        Returns the bnusd fee required for defining a vote.
+        """
+        return self._bnusd_vote_definition_fee.get()
+    
+    @external
+    @only_owner
+    def setBalnVoteDefinitionCriteria(self, percentage: str) -> None:
+        """
+        Set the minimum percentage of baln's total supply which a user must have staked
+        in order to define a vote.
+
+        Parameters:
+        percentage - Percentage of total baln needed to define a vote. E.g. "1.5" -> 1.5%.
+        """
+        decimal = float(percentage) / 100
```

Comment:
Its better not to use float in the score completely as float calculation will get rejected in the score.
For the percentage we have been implementing it in terms of POINTS(= 10000) in loan score maybe you have take the reference from loans score.

Reply:
Got it. I'll take a look at the loan contract and make those changes.

Reply:
I'm not entirely sure what you mean by rejected? Audit? I talked to Spl3en (one of the contract auditors) regarding contracts failing contract audit because of float operations (I found it a bit wierd since float operations are entirely deterministic, so it should not be a problem). He confimed that contracts do not fail audit because of float operations, except when it's used as a return value for @external(readonly=true) functions or as a key dict type.

With that said. I'll of course change the code if you want it represented only with integers.

Reply:
Yeah, since we were implementing percentage in terms of points all over the system i wanted it to be that way. And thanks for those insights that might  be useful in future.

Reply:
That makes perfect sense. Thanks for reviewing 👍

Reply:
Pushed some new changes to accommodate Dans review. Github says "Skyvell dismissed simusud’s stale review via 0fff5f1 3 hours ago", not sure why it said that; I did not dismiss anything :)